src/components/InlineSpinner/InlineSpinner.tsx

```diff
@@ -50,6 +50,12 @@ const InlineSpinner: React.FC<InlineSpinnerProps> = styled.div<InlineSpinnerProp
     ${compose(margin, sizeVariant)}
 `;
 
+const InlineSpinner = () => (
+    <span>
```

Comment:
do we understand why we need this span? Semantically it looks strange `span > div`

Reply:
It seems to be related to the way inline display works. But I am not able to articulate a full response over this _(That's why I add the note I added on the PR description._ Rather I will point you to:
- https://codesandbox.io/s/button-spinner-issue-wnmgv?file=/src/App.tsx
- https://mytaxi.slack.com/archives/CCWTB4C6L/p1620738413014800